console/frontend/src/main/frontend/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,378 @@
+    title: 'Edit Schedule',
+    data: {
+      breadcrumbs: 'Scheduler > Edit Schedule'
+    },
+  },
+  {
+    path: 'environment-variables',
+    component: EnvironmentVariablesComponent,
+    title: 'Environment Variables',
+    data: {
+      breadcrumbs: 'Environment Variables'
+    }
+  },
+  {
+    path: 'jdbc/execute-query',
+    component: JdbcExecuteQueryComponent,
+    title: 'Execute JDBC Query',
+    data: {
+      breadcrumbs: 'JDBC > Execute Query'
+    }
+  },
+  {
+    path: 'jdbc/browse-tables',
+    component: JdbcBrowseTablesComponent,
+    title: 'Browse JDBC Tables',
+    data: {
+      breadcrumbs: 'JDBC > Browse Tables'
+    }
+  },
+  {
+    path: 'security-items',
+    component: SecurityItemsComponent,
+    data: {
+      pageTitle: 'Security Items',
+      breadcrumbs: 'Security Items'
+    }
+  },
+  {
+    path: 'connections',
+    component: ConnectionsComponent,
+    title: 'Connection Overview',
+    data: {
+      breadcrumbs: 'Connection Overview'
+    }
+  },
+  {
+    path: 'inlinestores/overview',
+    component: InlinestoreComponent,
+    title: 'InlineStore Overview',
+    data: {
+      breadcrumbs: 'InlineStore Overview'
+    }
+  },
+  {
+    path: 'monitors',
+    component: MonitorsComponent,
+    title: 'Monitors',
+    data: {
+      breadcrumbs: 'Monitors'
+    }
+  },
+  {
+    path: 'monitors/:monitor/triggers/:trigger',
+    component: MonitorsAddEditComponent,
+    title: 'Edit Trigger',
+    data: {
+      breadcrumbs: 'Monitors > Triggers > Edit'
+    },
+  },
+  {
+    path: 'monitors/:monitor/triggers/new',
+    component: MonitorsAddEditComponent,
+    title: 'Add Trigger',
+    data: {
+      breadcrumbs: 'Monitors > Triggers > Add'
+    },
+  },
+  {
+    path: 'ibisstore-summary',
+    component: IbisstoreSummaryComponent,
+    title: 'Ibisstore Summary',
+    data: {
+      breadcrumbs: 'JDBC > Ibisstore Summary'
+    }
+  },
+  {
+    path: 'liquibase',
+    component: LiquibaseComponent,
+    title: 'Liquibase Script',
+    data: {
+      breadcrumbs: 'JDBC > Liquibase Script'
+    }
+  },
+  {
+    path: 'customView/:name',
+    component: IframeCustomViewComponent,
+    title: "Custom View",
+    data: {
+      breadcrumbs: 'Custom View',
+      iframe: true
+    },
+  },
+  {
+    path: 'testing/larva',
+    component: IframeLarvaComponent,
+    title: 'Larva',
+    data: {
+      breadcrumbs: 'Testing > Larva',
+      iframe: true
+    },
+  },
+  {
+    path: 'testing/ladybug',
+    component: IframeLadybugComponent,
+    title: 'Ladybug',
+    data: {
+      breadcrumbs: 'Testing > Ladybug',
+      iframe: true
+    },
+  },
+  {
+    path: 'testing/ladybug-beta',
+    component: IframeLadybugBetaComponent,
+    title: 'Ladybug (beta)',
+    data: {
+      breadcrumbs: 'Testing > Ladybug (beta)',
+      iframe: true
+    },
+  },
+  /* {
+    path: 'empty_page',
+    templateUrl: "js/app/views/empty/empty_page.html",
+    title: 'Empty Page'
+  }, */
```

Comment:
Is this still needed?

Reply:
Not really, its always been there but I commented it out since I didn't need it and it hasn't been used for anything